src/main/java/software/bernie/techarium/integration/ModIntegrations.java

```diff
@@ -27,6 +28,15 @@
     public static final Integration.Wrapper<ThermalCultivationIntegration> THERMAL_CULTIVATION = Integration.Wrapper.of("thermal",
             ThermalCultivationIntegration::new).registerSelf();
 
+    static {
+        System.out.println("preTheOneProbe");
+    }
+
+    public static final Integration.Wrapper<TheOneProbeIntegration> THE_ONE_PROBE = Integration.Wrapper.of("theoneprobe",
+            TheOneProbeIntegration::new).registerSelf();
+    static {
+        System.out.println("postTheOneProbe");
```

Comment:
Probably should use a Logger and also make the logging messages better

Reply:
They were for debugging, missed to remove them apperently gonna fix that in hwyla